webextensions/api/userScripts_legacy.json

```
@@ -0,0 +1,132 @@
+{
+  "webextensions": {
+    "api": {
+      "userScripts_legacy": {
+        "__compat": {
+          "description": "userScripts (Legacy)",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "68",
+              "notes": [
+                "Only available for use in extensions using Manifest V2.",
+                "An alternative version of this API is available for use with Manifest V3. See [`userScripts`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/userScripts)."
+              ]
+            },
+            "firefox_android": "mirror",
+            "opera": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror"
+          }
+        },
+        "RegisteredUserScript": {
+          "__compat": {
+            "description": "RegisteredUserScript (Legacy)",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "68",
+                "notes": "For extension using Manifest V3, see [`userScripts.RegisteredUserScript`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/userScripts/RegisteredUserScript)."
+              },
+              "firefox_android": "mirror",
+              "opera": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror"
+            }
+          },
+          "unregister": {
+            "__compat": {
+              "description": "unregister (Legacy)",
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "68",
+                  "notes": "For extension using Manifest V3, see [`userScripts.unregister`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/userScripts/unregister)."
```

On rereading - I can only agree!
```suggestion
                  "notes": "Only available for use in extensions using Manifest V2. For extension using Manifest V3, see [`userScripts.unregister`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/userScripts/unregister)."
```